qa/tasks/cephfs/test_scrub_checks.py

```diff
@@ -307,7 +307,7 @@ def tell_command(self, mds_rank, command, validator):
 
         success, errstring = validator(jout, 0)
         if not success:
-            raise AsokCommandFailedError(command, rout, jout, errstring)
+            raise AsokCommandFailedError(command, success, jout, errstring)
```

Comment:
should pass "0" instead of "success" as the return code. as success here is "False" instead of a number.

Reply:
done